pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -68,7 +65,14 @@ def __init__(self, logger, handler, socket_path, platform_kind, namespace=None,
 
         # call init context
         if hasattr(entrypoint_module, 'init_context'):
-            getattr(entrypoint_module, 'init_context')(self._context)
+            try:
+                getattr(entrypoint_module, 'init_context')(self._context)
+            except:
+                self._logger.warn_with('Exception raised while running init_context')
+                raise
+
+        # replace the default output with the process socket
+        self._logger.set_handler('default', self._processor_sock_wfile, nuclio_sdk.logger.JSONFormatter())
```

Comment:
why moving it here? wouldn't it affect the Context object being initialized with this logger?

Reply:
moved it here specifically for the case where the wrapper fails while running `init_context`

Reply:
I moved it because if an exception happened while running `init_context()` we wouldn't get it logged otherwise